src/core/Akka.Remote.Tests.Performance/Transports/RemoteMessagingThroughputSpecBase.cs

```diff
@@ -123,19 +124,16 @@ public void TwoWay(BenchmarkContext context)
                 _remoteEcho.Tell("foo", _receiver); // send a remote message
                 ++i;
             }
-            _resetEvent.Wait(1000); //wait up to a second
+            _resetEvent.Wait();
         }
 
         [PerfCleanup]
-        public void Cleanup()
+        public virtual void Cleanup()
         {
-            if (!_resetEvent.IsSet)
-                _resetEvent.Wait(TimeSpan.FromSeconds(10)); // wait indefinitely until the event is set
             _resetEvent.Dispose();
             System1.Shutdown();            
-            System1.AwaitTermination(TimeSpan.FromSeconds(2));
             System2.Shutdown();
-            System2.AwaitTermination(TimeSpan.FromSeconds(2));
+            Task.WaitAll(System1.TerminationTask, System2.TerminationTask);
```

Comment:
I changed the way we wait for the `ActorSystem`s to exit here also - found that this also helped, although there are still some issues with the `TestTransport` that need to be resolved for squeaky-clean benchmarking.